vault/checksum_map.jinja

```diff
@@ -14,5 +14,6 @@
   '0.6.5': '11555fb56cf41cf2dfdd0e53a47fe020b7bbc15419e10af03ebcd566434b6e13f845d592202b1d9c660529abc331c76cfc649f3703e5fff665791dd031af385b',
   '0.7.3': 'bf4450027d068a03c8181d2da8411395be777474d6c405e5a880a52fdd3cf2eac6a0925fa5d172ec3325f564045bf765db5e2c0c931562329bec7c8fd9e620b4',
   '0.8.3': '02e3c868499941e3747177ab91e67a91961eb08aa803de67d8b7b43709b10f575624e90a1c459b41d0eebca6ce771794d6516a737bed79cc336dc20ef94ff97b',
+  '0.10.3': 'ffec1c201f819f47581f54c08653a8d17ec0a6699854ebd7f6625babb9e290ed',
```

Comment:
the sha should be `b51af5208eb05e54fbe895e6dbf26633b96686d8278311091d5fdf1a2e58517bf17f0f5a2be103237456a57bde6a0f2b3b42e82a93bc7aeeffc9b91bb3dda8db`